_translations/en.yml

```diff
@@ -285,23 +285,23 @@ en:
-    walletkeepkey: "KeepKey is a hardware wallet that makes bitcoin security simple. When you entrust KeepKey with your money, every bitcoin transaction you make must be reviewed and approved via its OLED display and confirmation button."
+    wallettrezor: "Trezor is a hardware wallet providing a high level of security without sacrificing convenience. Unlike cold storage, Trezor is able to sign transactions while connected to an online device."
+    walletcoldcard: "Coldcard is an easy to use, secure, open-source and affordable hardware wallet that is easy to back up via an encrypted microSD card. Your private key is stored in a dedicated security chip. MicroPython software design allows you to make changes."
+    walletkeepkey: "KeepKey is a hardware wallet that makes bitcoin security more simple. When you entrust KeepKey with your money, every bitcoin transaction you make must be reviewed and approved via its OLED display and confirmation button."
     walletbitpay: "BitPay Wallet is the HD-multisignature wallet originally built to secure BitPay's funds. BitPay supports multiple personal and shared wallets, testnet, and the full Payment Protocol. A private <a href=\"https://github.com/bitpay/bitcore-wallet-service\">BWS</a> node can be used for enhanced security and privacy."
     walletledgernanos: "Ledger Nano S is a secure Bitcoin hardware wallet. It connects to any computer through USB and embeds a built-in OLED display to double-check and confirm each transaction with a single tap on its buttons."
-    walletbitbox: "BitBox is a minimalist hardware wallet from Switzerland that emphasizes security and privacy. Features include fully offline &amp; simplified backup, plausible deniability, multi-sig support, a native desktop app, and a mobile app for verification &amp; 2FA."
-    walletbither: "Bither is a simple and secure wallet on many platforms. With special designed Cold/Hot modes, user can easily get both safety and simplicity. Bither's XRANDOM uses different entropy sources to generate true random number for users. Also with HDM, users can have HD's advantages and Multisig's security."
+    walletbitbox: "BitBox is a minimalist hardware wallet from Switzerland that emphasizes security and privacy. Features include fully offline &amp; simplified backup, plausible deniability, multisig support, a native desktop app, and a mobile app for verification &amp; 2FA."
+    walletbither: "Bither is a simple and secure wallet on many platforms. With special designed cold/hot modes, user can more easily get both safety and simplicity. Bither's XRANDOM uses different entropy sources to generate true random number for users. Also with HDM, users can have HD's advantages and multisig's security."
```

Comment:
As above, recommend removing "more" for consistency and simplicity.

Reply:
a1c31988